ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -2896,3 +2894,41 @@ export const testMissingLayouts = (): Promise<boolean> =>
       ],
     });
   });
+
+export const testProgressIndicator = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    await whisper.create({
+      label: 'Did progress indicator show the percentage? ',
+      onClose: () => {
+        console.debug('closed');
+      },
+      components: [
+        {
+          type: WhisperComponentType.Progress,
+          determinate: 42,
+        },
+        resolveRejectButtons(resolve, reject, 'Yes', 'No', true),
+      ],
+    });
+  });
+
+export const testProgressIndicatorIndefinite = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    await whisper.create({
+      label: 'Did progress indicator have indefinite animation? ',
+      onClose: () => {
+        console.debug('closed');
+      },
+      components: [
+        {
+          type: WhisperComponentType.Progress,
+          size: StyleSize.Medium,
```

Comment:
For the purposes of this test, could you try this out with all the different sizes and shapes?